frontend/scss/components/atoms/pill.scss

```diff
@@ -48,4 +48,11 @@
     box-shadow: none;
     background-color: color('wild-sand');
   }
+
+  &-meta, &-meta:hover {
```

Comment:
The `:hover` selector is unnecessary as it doesn't have any different styles to the unhovered state. Also the meta pills are not clickable.